spec.html

```diff
@@ -30851,11 +30851,13 @@ <h1>Runtime Semantics: RegExpInitialize ( _obj_, _pattern_, _flags_ )</h1>
             1. If _F_ contains any code unit other than `"g"`, `"i"`, `"m"`, `"s"`, `"u"`, or `"y"` or if it contains the same code unit more than once, throw a *SyntaxError* exception.
             1. If _F_ contains `"u"`, let _BMP_ be *false*; else let _BMP_ be *true*.
             1. If _BMP_ is *true*, then
```

Comment:
kind of seems silly to have this "BMP" variable when it could just inline the "F contains u" part? (predates this PR, ofc)

Reply:
Yeah, it's been like that since it was introduced.

> it could just inline the "F contains u" part?

You'd have to either use *does not contain* or else swap the if-arms, and I didn't know which would be preferable. (Plus it's not essential to the point of the PR, and I'm never sure how much of those to include.)

Reply:
@ljharb: I'm planning to fix the `_BMP_` thing in my follow-up to PR #1464.